core/src/main/java/hudson/Functions.java

```diff
@@ -233,6 +233,8 @@ public static void initPageVariables(JellyContext context) {
 
         // The path starts with a "/" character but does not end with a "/" character.
         context.setVariable("rootURL", rootURL);
+        Jenkins jenkins = Jenkins.getActiveInstance();
```

Comment:
If my expectation is true, there will be angry Jenkins instead of the startup page